AudioSinkExample.xcodeproj/project.pbxproj

```diff
@@ -0,0 +1,375 @@
+				8A6C2E5E1FAD3EAC008EB058 /* Utils */,
+				8A6C2E491FAD3AD2008EB058 /* Products */,
+			);
+			sourceTree = "<group>";
+		};
+		8A6C2E491FAD3AD2008EB058 /* Products */ = {
+			isa = PBXGroup;
+			children = (
+				8A6C2E481FAD3AD2008EB058 /* AudioSinkExample.app */,
+			);
+			name = Products;
+			sourceTree = "<group>";
+		};
+		8A6C2E4A1FAD3AD2008EB058 /* AudioSinkExample */ = {
+			isa = PBXGroup;
+			children = (
+				8A6C2E4B1FAD3AD2008EB058 /* AppDelegate.swift */,
+				8A6C2E621FAD48DB008EB058 /* AudioSinks */,
+				8A6598E41FB13ED6005A8849 /* RecordingsViewController.swift */,
+				8A6C2E4D1FAD3AD2008EB058 /* ViewController.swift */,
+				8A6C2E4F1FAD3AD2008EB058 /* Main.storyboard */,
+				8A6C2E521FAD3AD2008EB058 /* Assets.xcassets */,
+				8A6C2E541FAD3AD2008EB058 /* LaunchScreen.storyboard */,
+				8A6C2E571FAD3AD2008EB058 /* Info.plist */,
+			);
+			path = AudioSinkExample;
+			sourceTree = "<group>";
+		};
+		8A6C2E5E1FAD3EAC008EB058 /* Utils */ = {
+			isa = PBXGroup;
+			children = (
+				8A6C2E5F1FAD3EAC008EB058 /* Settings.swift */,
+				8A6C2E601FAD3EAC008EB058 /* SettingsTableViewController.swift */,
+				8A6C2E611FAD3EAC008EB058 /* Utils.swift */,
+			);
+			path = Utils;
+			sourceTree = "<group>";
+		};
+		8A6C2E621FAD48DB008EB058 /* AudioSinks */ = {
+			isa = PBXGroup;
+			children = (
+				8A7FE3B51FAD6409003A4C41 /* AudioSinks-Bridging-Header.h */,
+				8A6C2E631FAD48DB008EB058 /* ExampleAudioRecorder.h */,
+				8A6C2E641FAD48DB008EB058 /* ExampleAudioRecorder.m */,
+				8A6C2E651FAD48DB008EB058 /* ExampleSpeechRecognizer.h */,
+				8A6C2E661FAD48DB008EB058 /* ExampleSpeechRecognizer.m */,
+			);
+			path = AudioSinks;
+			sourceTree = "<group>";
+		};
+/* End PBXGroup section */
+
+/* Begin PBXNativeTarget section */
+		8A6C2E471FAD3AD2008EB058 /* AudioSinkExample */ = {
+			isa = PBXNativeTarget;
+			buildConfigurationList = 8A6C2E5A1FAD3AD2008EB058 /* Build configuration list for PBXNativeTarget "AudioSinkExample" */;
+			buildPhases = (
+				8A6C2E441FAD3AD2008EB058 /* Sources */,
+				8A6C2E451FAD3AD2008EB058 /* Frameworks */,
+				8A6C2E461FAD3AD2008EB058 /* Resources */,
+			);
+			buildRules = (
+			);
+			dependencies = (
+			);
+			name = AudioSinkExample;
+			productName = AudioSinkExample;
+			productReference = 8A6C2E481FAD3AD2008EB058 /* AudioSinkExample.app */;
+			productType = "com.apple.product-type.application";
+		};
+/* End PBXNativeTarget section */
+
+/* Begin PBXProject section */
+		8A6C2E401FAD3AD2008EB058 /* Project object */ = {
+			isa = PBXProject;
+			attributes = {
+				LastSwiftUpdateCheck = 0900;
+				LastUpgradeCheck = 0900;
+				ORGANIZATIONNAME = "Twilio Inc.";
+				TargetAttributes = {
+					8A6C2E471FAD3AD2008EB058 = {
+						CreatedOnToolsVersion = 9.0;
+						ProvisioningStyle = Automatic;
+					};
+				};
+			};
+			buildConfigurationList = 8A6C2E431FAD3AD2008EB058 /* Build configuration list for PBXProject "AudioSinkExample" */;
+			compatibilityVersion = "Xcode 8.0";
```

Comment:
This doesn't match with the README requirement.

Reply:
I guess I will need to try it on Xcode 8.3.3 to be sure. I'm using APIs that are only available in the iOS 11.0 SDK, but they are checked behind an availability flag.